Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -942,7 +939,7 @@ - (void)callClose {
     if (self.isInitialized) {
         self.isInitialized = NO;
 
-        if ([self hasSession] && ![self.requestQueue containsClose]) {
+        if (![self.requestQueue containsClose]) {
```

Comment:
Hmm, this was an accidental / "partial" remove. This was really just a band aid to figure out *why* session id was being lost (the root of which is, I believe, because defaults were being cleared). I believe this PR will fix that so this check shouldn't be necessary, but I will revert this for now until I've had more time to prove that.